File: 1. Node/Node.py
class Node:
    def __init__ (self,data):
        self.data = data
        self.next = None
    
    def setNextNode(self,node):
        self.next = node
    
    def getNodeNext(self):
        return self.next
    
first_node = Node("I am first node")
second_node = Node("I am second node")
third_node = Node("I am Third node")

first_node.setNextNode(second_node)
second_node.setNextNode(third_node)

current_node = first_node

while current_node is not None:
    print(current_node.data)
    current_node = current_node.getNodeNext()

File: 1. Node/Node.java
class Node{
  public String data;
  public Node next;

  public Node(String data){
    this.data = data;
    this.next = null;
  }

  public void setNextNode(Node node){
    this.next = node;
  }

  public Node getNextNode(){
    return this.next;
  }

  public static void main(String []args){
    Node firstNode = new Node("I am first node");
    Node secondNode = new Node("I am second node");
    Node thirdNode = new Node("I am Third node");

    firstNode.setNextNode(secondNode);
    secondNode.setNextNode(thirdNode);

    Node current_node = firstNode;

    while(current_node != null){
      System.out.println(current_node.data);
      current_node = current_node.getNextNode();
    }
  }

}

// https://www.codecademy.com/learn/paths/pass-the-technical-interview-with-java/tracks/getting-started-with-data-structures-java/modules/nodes-java/cheatsheet
